static/src/javascripts/projects/common/modules/analytics/interaction-tracking.js

```diff
@@ -0,0 +1,106 @@
+// @flow
+import mediator from 'lib/mediator';
+import { session } from 'lib/storage';
+import {
+    trackSamePageLinkClick as trackSamePageLinkClickGoogle,
+    trackNonClickInteraction as trackNonClickInteractionGoogle,
+    trackExternalLinkClick as trackExternalLinkClickGoogle,
+    trackSponsorLogoLinkClick as trackSponsorLogoLinkClickGoogle,
+} from 'common/modules/analytics/google';
+import { catchErrorsWithContext } from 'lib/robust';
+
+const NG_STORAGE_KEY = 'gu.analytics.referrerVars';
+let loc = document.location;
+
+type Spec = {
+    validTarget: boolean,
+    target: any,
+    sameHost: boolean,
+    samePage: boolean,
+    tag: string,
+};
+
+const isSponsorLogoLinkClick = (target: Object) =>
+    target.hasAttribute('data-sponsor');
+
+// used where we don't have an element to pass as a tag, eg. keyboard interaction
+const trackNonClickInteraction = (actionName: string) => {
```

Comment:
is `trackNonClickInteraction` redundant indirection here?

Reply:
It is redundant. I think it's because the idea was to dispatch to all the tracking services you wanted to support, but we now only support GA so it just forwards directly to that only. I didn't change how it worked because I wanted to preserve the intention of the developer that it could be used for dispatching, and because I didn't want to refactor the rest of this code to be GA-Only to match, under an es6 conversion pr.